src/components/ModalResetAllData.js

```diff
@@ -75,8 +68,9 @@ class ModalResetAllData extends React.Component {
     }
 
     if (!forgotPassword) {
-      // Password was informed and it is incorrect
-      const correctPassword = await this.props.wallet.checkPassword(password);
+      // Password was informed and it is incorr
+      const wallet = getGlobalWallet();
+      const correctPassword = await wallet.checkPassword(password);
```

Comment:
```suggestion
      const { checkPassword } = getGlobalWallet();
      const correctPassword = await checkPassword(password);
```

Reply:
It's not possible to use this syntax with a HathorWallet method: it crashes the application.